Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -268,7 +277,10 @@ bool CBoot::SetupWiiMemory(IOS::HLE::IOSC::ConsoleType console_type)
     INFO_LOG(BOOT, "Using serial number: %s", serno.c_str());
   }
 
-  std::string model = "RVL-001(" + region_setting.area + ")";
+  std::string model = "RVL-001(";
+  model += region_setting.area;
+  model += ')';
```

Comment:
(Why) Does this make a difference? I mean the splitting into multiple lines; not the use of `')'` (`char`) vs. `")"` (`char*`)

Reply:
You can't write `"RVL-001(" + region_setting.area`, because there is no `+` operator that takes two `const char*`. The way it's currently written, `"RVL-001("` gets converted to `std::string` by the first line, making it work.

Reply:
With that in mind, if you want to write it on one line, you can write it as `std::string("RVL-001(") + region_setting.area + ')'` (or `"RVL-001(" + std::string(region_setting.area) + ')'`).

Reply:
Oh, right, the type of `region_setting.area` also changed. Nvm this one then.